openmdao/docs/openmdao_book/features/experimental/jax_explicitcomp_api.ipynb

```diff
@@ -0,0 +1,295 @@
+    "\n",
+    "3. By default your component will jit the compute_primal method. If for some reason you don't want this, then\n",
+    "you can set self.options['use_jit'] to False.\n",
+    "\n",
+    "4. For a typical component like `SellarDis2` below, that's it.  You can skip step 5.\n",
+    "\n",
+    "5. However, if your compute_primal depends on variables that are 'static', i.e., they don't change during computation of derivatives, you'll need to define a `get_self_statics` method on your component that returns a tuple containing all of the static variables that your compute_primal method depends on, excluding discrete input variables.  The returned tuple must be hashable.  If these static values ever change, jax will recompile the `compute_primal` function, assuming 'use_jit' is True.  In `SellarDis1` below, there is one static attribute, `self.staticvar`, and one static option variable, `self.options['scaling_ref']`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class SellarDis1(om.JaxExplicitComponent):\n",
+    "    def initialize(self):\n",
+    "        # Added this option to this model to demonstrate how having options\n",
+    "        # requires special care when using jit. See comments below\n",
+    "        self.options.declare('static_opt', types=(float,), default=1.)\n",
+    "        self.staticvar = 1.\n",
+    "\n",
+    "    def setup(self):\n",
+    "        # Global Design Variable\n",
+    "        self.add_input('z', val=jnp.zeros(2))\n",
+    "\n",
+    "        # Local Design Variable\n",
+    "        self.add_input('x', val=0.)\n",
+    "\n",
+    "        # Coupling parameter\n",
+    "        self.add_input('y2', val=1.0)\n",
+    "\n",
+    "        # Coupling output\n",
+    "        self.add_output('y1', val=1.0, lower=0.1, upper=1000., ref=0.1)\n",
+    "\n",
+    "    def setup_partials(self):\n",
+    "        self.declare_partials('*', '*')\n",
+    "\n",
+    "    # because our compute primal output depends on static variables, in this case self.staticvar\n",
+    "    # and self.options['static_opt'], we must define a get_self_statics method. This method must\n",
+    "    # return a tuple of all static variables. Their order in the tuple doesn't matter.  If your\n",
+    "    # component happens to have discrete inputs, do NOT return them here. Discrete inputs are passed\n",
+    "    # into the compute_primal function individually, after the continuous variables.\n",
```

Comment:
this says discrete inputs come "after the continuous variables", but in line 96 above it says "prior to the continuous input variables".